src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,280 @@
-      description: "Description forthcoming.",
-      bgColor: "linear-gradient(180deg, #19953E 0%, #196B5B 100%)",
-      ctaText: "Learn More",
+      titleText: "Download Data",
+      linkPath: "/data",
+      description:
+        "Download Fourth Assessment climate data in NetCDF, GeoTIFF and CSV formats.",
+      bgColor: "var(--card-gradient-03)",
     },
   ];
-  const cardHeight = 20;
-  const cardWidth = 18;
 
-  const updateLinksPlaceholders = [
-    {
-      title: "Update number one",
-      linkPath: dummyLink,
-    },
-    {
-      title: "Update number two",
-      linkPath: dummyLink,
-    },
-    {
-      title: "Update number three",
-      linkPath: dummyLink,
-    },
-  ];
+  export async function preload() {
+    const posts = await this.fetch(`blog.json`)
+      .then((r) => r.json())
+      .then((posts) => posts);
+    const events = await this.fetch(`events.json`)
+      .then((r) => r.json())
+      .then((events) => events);
+    const data = await this.fetch("index.json")
+      .then((r) => r.json())
+      .then((data) => data);
+    return {
+      posts,
+      events,
+      cardsData: cardsDataTemp,
+      revealNewHomepage: data.revealNewHomepage,
+    };
+  }
+</script>
+
+<script>
+  import { Button } from "carbon-components-svelte";
+  import { ArrowRight16 } from "carbon-icons-svelte";
+  import { Card, CardsContainer } from "~/components/cards";
+  import { Banner, SidebarRight } from "~/partials";
+  import { InlineNotification } from "carbon-components-svelte";
+  import InformationFilled32 from "carbon-icons-svelte/lib/InformationFilled32";
+
+  export let events;
+  export let posts;
+  export let cardsData;
+  export let revealNewHomepage;
+
+  const icons = [
+    "sun",
+    "rainfall",
+    "wildfire",
+    "snowflake",
+    "sea",
+    "streamflow",
+  ].map((name) => `img/icons/${name}.svg`);
+  const cardHeight = 18;
+  const cardWidth = 18;
 </script>
 
+<style lang="scss">
+  @import "scss/site/mixins/media-queries";
+
+  .alert :global(.bx--inline-notification) {
+    margin: 0;
+    max-width: 100%;
+    background: $teal-10;
+
+    &::before {
+      border: none;
+    }
+  }
+
+  .alert :global(.bx--inline-notification__text-wrapper) {
+    flex-direction: column;
+  }
+
+  .bx--inline-notification__subtitle {
+    margin-bottom: 0;
+  }
+
+  .btn-container {
+    margin-top: 2.5rem;
+    text-transform: uppercase;
+  }
+
+  // The following code creates a grid that has 1/3 and 2/3 column template
+  .one-two-column-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 33% 67%;
+    grid-gap: 2rem;
+    padding: 0;
+    margin: 0;
+  }
+
+  .ae-logo-box {
+    padding: 14px;
+
+    img {
+      display: block;
+      margin: auto;
+    }
+  }
+
+  .ae-logo-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-items: flex-start;
+
+    @include media("<=medium") {
+      padding-top: 24px;
```

Comment:
You won't need this `padding-top` for smaller screens once you make the changes to the parent grid (see comment above). The `grid-gap` will provide enough space between the elements. It might be useful to change the `span` of the div to 1 for smaller screens for a tighter layout. e.g. 

```
@include media("<=medium") {
  .ae-logo-text {
    grid-column : span 1;
  }
}
```